src/views/Map/Map.tsx

```
@@ -136,42 +152,45 @@ export function MapPage() {
+        searchBar={
+          featureFlags[AppFeature.SEARCH_BAR] &&
+          featureFlags[AppFeature.SEARCH_LOCATION] && <Search />
+        }
+        analytics={<Analytics />}
+        mapTitle={featureFlags[AppFeature.MAP_TITLE] && <MapTitle />}
+        // if EVENTS_LIST is enabled, we always have default feed
+        disasters={featureFlags[AppFeature.EVENTS_LIST] && <EventListPanel />}
+        layersAndLegends={<LayersAndLegends />}
+        legendsPanel={featureFlags[AppFeature.LEGEND_PANEL] && <LegendsPanel />}
+        currentEvent={featureFlags[AppFeature.CURRENT_EVENT] && <CurrentEvent />}
+        matrix={<></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)